<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileNameFormatWidget</class>
 <widget class="QWidget" name="FileNameFormatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>283</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="naming_group">
     <property name="title">
      <string>Naming options</string>
     </property>
     <layout class="QVBoxLayout" name="innerVerticalLayout">
      <item>
       <layout class="QHBoxLayout" name="innerHorizontalLayout">
        <item>
         <widget class="LineTextEdit" name="naming">
          <property name="toolTip">
           <string>&lt;p&gt;Tokens start with %, for example: %artist %album %title &lt;/p&gt;

&lt;p&gt;If you surround sections of text that contain a token with curly-braces, that section will be hidden if the token is empty.&lt;/p&gt;</string>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="insert">
          <property name="text">
           <string>Insert...</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="replace_the">
        <property name="text">
         <string>Ignore &quot;The&quot; in artist names</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="replace_spaces">
        <property name="text">
         <string>Replaces spaces with underscores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="replace_ascii">
        <property name="text">
         <string>Restrict to ASCII characters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="overwrite">
        <property name="text">
         <string>Overwrite existing files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mark_as_listened">
        <property name="text">
         <string>Mark as listened</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineTextEdit</class>
   <extends>QTextEdit</extends>
   <header>widgets/linetextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>naming</tabstop>
  <tabstop>insert</tabstop>
  <tabstop>replace_the</tabstop>
  <tabstop>replace_spaces</tabstop>
  <tabstop>replace_ascii</tabstop>
  <tabstop>overwrite</tabstop>
  <tabstop>mark_as_listened</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
